_posts/2021-11-18-real-time-and-historical-ad.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Anomaly Detection for Historical Data and Real-time Streaming"
+authors:
+  - ohltyler
+  - bpavani
+date: 2021-11-18 09:00:00 -0700
+categories:
+  - technical-post
+twittercard:
+  description: "An overview on configuring and creating detectors for real-time and historical anomaly detection in OpenSearch."
```

Comment:
It might be better to use the first couple of sentences instead of this summary.